doc/nrf/libraries/zigbee/lib_zigbee_shell.rst

```diff
@@ -0,0 +1,662 @@
+﻿.. _lib_zigbee_shell:
+
+Zigbee shell
+############
+
+The Zigbee shell library implements a set of Zigbee shell commands that allow you to control the device behavior and perform operations in the Zigbee network, such as steering the network, discovering devices, and reading the network properties.
+Enabling a device to work with these commands simplifies testing and debugging of an existing network and reduces the amount of reprogramming operations.
+
+.. _zigbee_shell_extending_samples:
+
+Adding support for Zigbee shell commands
+****************************************
+
+The Zigbee shell commands are implemented using Zephyr's :ref:`zephyr:shell_api` interface.
+By default, it uses the UART backend.
+To change this and other Zephyr's shell settings (for example, the prompt or the maximum amount of accepted command arguments), check the documentation page in Zephyr.
```

Comment:
@edmont , given your recent changes here:
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_logging.html#shell
Can you have a look at this PR and see if you have any concerns about Zephyr Shell mentions?